config/locales/subscriber_authentication.yml

```diff
@@ -1,18 +1,18 @@
 en:
   subscriber_authentication:
-    heading: Manage your subscriptions
+    heading: Manage your email subscriptions
     sign_in:
-      description: We’ll send you an email with a link in to manage your subscriptions.  There may be a delay of up to 15 minutes before you receive this email.
+      description: We’ll send you an email with a link in to manage your subscriptions.
       email_input:
         label: What’s your email address?
       missing_email: "Please enter your email address."
       invalid_email: "This doesn’t look like a valid email address – check you’ve entered it correctly."
       email_error:
-        title: We weren’t able to sign you in
+        title: We could not sign you in
```

Comment:
Is ```sign you in``` the right concept here?
If a user does not have an account, what are they signing in to?

Reply:
@vanitabarrett do you have any thoughts on this?

Reply:
@karlchillmaid I guess from a user point of view this process is similar to signing in - they're signing in to the 'manage subscriptions' area of GOV.UK. 

But it does seem like this might not be the right wording for this particular error message because at this point they're just trying to validate their email address. Even if they get past this step, they still need to click a link in an email to actually sign them in.

<img width="693" alt="Screenshot 2020-04-14 at 09 53 58" src="https://user-images.githubusercontent.com/29889908/79205468-e1824980-7e35-11ea-89b5-3f6804c80531.png">

Perhaps the error message could just be about the email address not being valid?

Reply:
@vanitabarrett Working on error messages at the moment. We may want to put them through as a separate PR if it looks like they're going to hold this one up.

Reply:
@vanitabarrett I've made a couple of small changes to the errors. This addresses the 'sign in' issue.

There's more we can do to improve these errors, but I think we'll need more input from design. We can address that later if we think it's a priority.